guides/common/modules/con_prerequisites-for-configuring-project-with-keycloak-quarkus-authentication.adoc

```diff
@@ -12,9 +29,13 @@ ifndef::orcharhino[]
 For more information about configuring a different context path, see the {RHDocsBaseURL}red_hat_build_of_keycloak/24.0/html-single/server_guide/index#reverseproxy-different-context-path-on-reverse-proxy[_{RHBK} Administration Guide_].
 endif::[]
 ====
-* If the certificates or the CA are self-signed, add them to the end-user certificate truststore.
-* A {keycloak} realm created for {Project} user accounts, for example `_{Project}_Realm_`.
-* Users imported or added to {keycloak}.
++
+If `keycloak-httpd-client-install` version 1.3 or later is installed, your {keycloak} server does not need to be initialized with the `--http-relative-path=/auth` context path.
+* Your {keycloak} server uses HTTPS instead of HTTP.
+* If the certificates or the CA are self-signed, they have been added to the end-user certificate truststore.
```

Comment:
suggestion (non-blocking): Use active voice.

* Who adds what where? I think you can copy this more of less from `guides/common/modules/proc_configuring-a-host-for-registration.adoc`.

Reply:
This might be a bit trickier because with multiple systems (Foreman, Keycloak), you might also have different admins. So the "who" might not be the same person, which made me think passive was more appropriate. (I remember checking the style guides when I was writing this, passive voice should be fine in these situations.)